doc/changes/changes_2.1.1.md

```diff
@@ -1,36 +1,39 @@
 # Virtual Schema for SQLServer 2.1.1, released 2023-??-??
 
-Code name:
+Code name: Dependency Upgrade on Top of 2.1.0
 
 ## Summary
 
-## Features
+This release fixes vulnerability CVE-2023-42503 in transitive dependency to `org.apache.commons:commons-compress` via `exasol-testcontainers` by updating dependencies.
```

Comment:
If it is a test-dependency, then please explicitly state that production code was not affected.